module1/homework/grayscaleImages/compression.hpp

```diff
@@ -0,0 +1,14 @@
+#pragma once
+
+#include <array>
+#include <utility>
+#include <vector>
+
+#include <cstdint>
```

Comment:
Why cstdint is included in such weird way instead of alphabetically with the rest.

Reply:
because we did it at last moment :)